amethyst_ui/src/shaders/vertex.glsl

```diff
@@ -0,0 +1,27 @@
+// TODO: Needs documentation.
+
+#version 150 core
+
+layout (std140) uniform VertexArgs {
```

Comment:
I don't know what `std140` does, could you add a comment, please?

Reply:
Ah I see. What's the default behavior? I can't recall myself writing anything other than

```c
struct MyStruct {
    int a;
}

uniform MyStruct my_struct;
```